### PR TITLE
fix(init): change initial site URL to one that is safe for the future

### PR DESCRIPTION
**Pull request by tats-u**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

The current initial template specifies `https://your-docusaurus-test-site.com` for the site URL. This is used for OGP  contents. *Everyone can reserve this domain* and an organization (or person) related to casino in Turkey has already done.

https://www.virustotal.com/gui/url/e8aeabbb3b02e8c207176c74dbf41fbf807fbf4be6f45c7f6ff9a139417ec181/details
https://otx.alienvault.com/indicator/domain/your-docusaurus-test-site.com

[RFC 2606](https://datatracker.ietf.org/doc/html/rfc2606#section-2) reserves the `.example` domain for illustrative purposes. (See the Section 2) We have to use this instead. (`.test` or `example.{com,org,net}`are also OK)

### Test links

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

### CODE REVIEW

File: packages/create-docusaurus/templates/classic/docusaurus.config.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const config = {
   favicon: 'img/favicon.ico',
 
   // Set the production url of your site here
-  url: 'https://your-docusaurus-test-site.com',
+  url: 'https://your-docusaurus-site.example',
```
Comment (Josh-Cena) [inline packages/create-docusaurus/templates/classic/docusaurus.config.js:17]: Maybe use `.example.com`? Seems more unsurprising.

Comment (tats-u) [reply packages/create-docusaurus/templates/classic/docusaurus.config.js:17]: @Josh-Cena Not so terrible, but it looks longer than `.example`.
A candidate is `your-docusaurus-site.example.com` or `docusaurus-site.example.com`.

https://example.com & https://www.example.com are resolved to an example page by IANA, but pages for the other subdomains  don't seem to be prepared.

Comment (Josh-Cena) [reply packages/create-docusaurus/templates/classic/docusaurus.config.js:17]: It's okay if the page is a broken page. And yes, I meant `your-docusaurus-site.example.com`.

Comment (tats-u) [reply packages/create-docusaurus/templates/classic/docusaurus.config.js:17]: Ok, I'll rename all of them.

Comment (tats-u) [reply packages/create-docusaurus/templates/classic/docusaurus.config.js]: @Josh-Cena Done.

**Review by Josh-Cena — COMMENTED** [review]
Almost good. Could you revert the changes in `examples/`? These are autogenerated. Also I think some fixtures have an extra trailing line break; I'm enabling the tests to see if they still pass.

**Comment by tats-u** [comment]
@Josh-Cena Are these all?
image

**Comment by Josh-Cena** [comment]
Yes

**Comment by tats-u** [comment]
@Josh-Cena Done

**Comment by tats-u** [comment]
Why did the Build Performance CI task fail?

**Review by Josh-Cena — APPROVED** [review]
Since the other tests are passing, this is fine. The build performance action is not very stable anyway.

**Comment by tats-u** [comment]
Thanks. We can cherry-pick this in the v2 branch.

**Comment by slorber** [comment]
> The current initial template specifies https://your-docusaurus-test-site.com/ for the site URL. This is used for OGP contents. Everyone can reserve this domain and an organization (or person) related to casino in Turkey has already done.

Damn, we live in a crazy world 😅 thanks for noticing this and fixing it

**Pull request merged**